designer/client/src/components/graph/node-modal/fragment-input-definition/settings/variants/DefaultVariant.tsx

```diff
@@ -29,7 +30,15 @@ export const DefaultVariant = ({ item, onChange, path, variableTypes, readOnly }
                 />
             </SettingRow>
             {/*<ValidationsFields path={path} onChange={onChange} item={item} variableTypes={variableTypes} />*/}
-            <InitialValue onChange={onChange} item={item} path={path} readOnly={readOnly} variableTypes={variableTypes} />
+            <InitialValue
+                onChange={onChange}
+                item={item}
+                path={path}
+                readOnly={readOnly}
+                variableTypes={variableTypes}
+                fieldsErrors={fieldsErrors}
+                fieldName={`$param.${item.name}.$initialValue`}
```

Comment:
Suggestion - probably out of scope - maybe we should extract these 'contractual' fieldNames and keep them in one place as they are on the BE?

Reply:
Ah, didn't notice that, this way is better :+1: